.env.example

```
@@ -1,3 +1,14 @@
+NODE_ENV=development
+
 # Database connection string
 # PostgreSQL: postgresql://USER:PASSWORD@HOST:PORT/DATABASE
 DATABASE_URL="postgresql://postgres:password@localhost:5432/daily_report_db"
+
+# JWT Configuration
+JWT_SECRET="your-super-secret-key-change-in-production"
+JWT_EXPIRES_IN="1h"
+
+# CORS
+CORS_ORIGIN="http://localhost:3000"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)